omniscidb/StringDictionary/StringDictionaryProxy.h

```diff
@@ -180,10 +180,12 @@ class StringDictionaryProxy {
   // in the std::map when new strings are added, but will change in a std::vector.
   using TransientMap = std::map<std::string, int32_t, std::less<>>;
 
-  const std::vector<std::string const*>& getTransientVector() const {
-    return transient_string_vec_;
-  }
+  std::vector<std::string> copyStrings() const;
 
+  // Iterate over transient strings, then non-transients.
+  void eachStringSerially(StringDictionary::StringCallback&) const;
```

Comment:
Is the iteration order significant?

Reply:
I don't think we provide any particular order. And this API is not actually used anyway. Method is moved to match the Dictionary API.